ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/ListArrayElement.java

```diff
@@ -5,7 +5,7 @@
 
 @Value
 @Builder
-public class CCDPartyArrayElement {
+public class ListArrayElement<T> {
```

Comment:
Can you rename it to be something like `CollectionElement` or `CCDCollectionElement`.